cogs/invoice/invoice.py

```diff
@@ -5,7 +5,7 @@
 
 from copy import copy
 from redbot.core import checks, commands, Config
-from redbot.core.utils.chat_formatting import box, bordered
```

Comment:
I take it that `invoice.py` just doesn't use `bordered` at all (i.e., redundant import)?